astropy/io/fits/tests/test_connect.py

```diff
@@ -414,6 +414,64 @@ def test_mask_str_on_read(self, tmp_path):
         tab = Table.read(filename, mask_invalid=False)
         assert tab.mask is None
 
+    def test_heterogeneous_VLA_tables(self, tmp_path):
+        """
+        Check the behaviour of heterogeneous VLA object.
+        """
+        filename = tmp_path / "test_table_object.fits"
+        msg = "Column 'col1' contains unsupported object types or " "mixed types: "
```

Comment:
```suggestion
        msg = "Column 'col1' contains unsupported object types or mixed types: "
```
This was a black precommit artifact...